benchmarks/dynamo/common.py

```diff
@@ -1978,12 +2084,14 @@ def run_n_iterations(self, mod, inputs):
             self.model_iter_fn(mod, inputs, collect_outputs=False)
         return self.model_iter_fn(mod, inputs, collect_outputs=True)
 
+    @torch._disable_dynamo(recursive=True)
```

Comment:
Wait why is this necessary? 

And if it's cuz we're doing eager adam now, should we only disable if the optim is not SGD?

Reply:
I want to enable SGD as well. Once I enable compiled this should go away. This is a better way to disable than torch patcher